src/components/PropsTable.tsx

```diff
@@ -102,8 +104,10 @@ export const PropsTable: React.FC<PropsTableProps> = ({ for: Component }) => {
           ))}
         </thead>
         <tbody>
-          {Object.keys(componentProps).map((key) => {
-            const { name, type, defaultValue, required } = componentProps[key]
+          {Object.keys(componentDocs.props).map((key) => {
```

Comment:
Could we use `Object.entries` here instead, to get the values without the extra step?

```tsx
{Object.entries(componentDocs.props).map(([key, {name, type, defaultValue, required}]) => {
   /* ... */
  }
}
```

Reply:
Ooof nice, yeah will do 👍